lib/eslint.js

```diff
@@ -798,7 +802,7 @@ module.exports = (function() {
                 impliedStrict: ecmaFeatures.impliedStrict,
                 ecmaVersion: ecmaVersion,
                 sourceType: currentConfig.parserOptions.sourceType || "script",
-                childVisitorKeys: espree.VisitorKeys,
+                childVisitorKeys: parser ? parser.VisitorKeys : espree.VisitorKeys,
                 fallback: "none"
```

Comment:
Not all parsers will have VisitorKeys, you need to use the espree VisitorKeys if not present.